src/createRouter.ts

```diff
@@ -202,7 +202,7 @@ const createRouter: RouterFactory = compose<RouterMixin, RouterOptions>({
 						return false;
 					}
 
-					const dispatched = (<Router> this).routes.some(route => {
+					const dispatched = this.routes.some((route: Route<Parameters>) => {
```

Comment:
We haven't typed `this` yet for `routing` and this change is because the original code `(<Router> this).routes.some` causes the compiler to report `Object is possibly 'undefined'` with `strictNullChecks` set to `true`